packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -242,7 +242,7 @@ class TextPainter {
         builder.pushStyle(text.style.getTextStyle(textScaleFactor: textScaleFactor));
       builder.addText(' ');
       _layoutTemplate = builder.build()
-        ..layout(const ui.ParagraphConstraints(width: double.infinity));
+        ..layout(ui.ParagraphConstraints(width: double.infinity));
```

Comment:
This should still be const.  Suspect it's just a bad merge

Reply:
That's what the analyzer tells me as well, but if I add const it gives an error. Should I just ignore it?

Reply:
Alternatively, you may not have had the latest engine available- this is a recent change